scaling_performance/cluster_limits.adoc

```diff
@@ -51,6 +51,10 @@ version or storage data format.
 | 10,000
 | 10,000
 
+| Number of pipeline strategy builds
```

Comment:
@mrsiano Is this defined build configurations?  concurrent running builds?  Probably best to qualify it.

Reply:
I believe it already done, thats what we've tested and tune for such a scale thought this plugin effort.

Reply:
I still think clarification might help.  Is it running builds or defined builds?

Reply:
the sample-pipline are defined builds right @gabemontero

Reply:
I wouldn't distinguish as "running vs. defined".

How about I describe the flow for pipeline strategy we employed in detail, and then you all can decide how to condense:
1) `oc start-build` of a pipeline strategy build config will create a build object in the new state
2) the openshift sync plugin running in our jenkins image will monitor for pipeline strategy build objects in the new state, read them into jenkins, and start a corresponding run in Jenkins (where the run is of the pipeline job created in jenkins when the pipeline strategy build config was created)

What @mrsiano measured for us was "how many builds could we reliably read into Jenkins at once, and still properly execute the runs for each new build".

What he discovered was:
a) at our default 512 Mi mem limit for the container, and 256Mi max heap for the JVM, we could read in 10K builds
b) when he doubled those 512/256 setting, we could read in 20K or 23K (somewhere around there)